From a97ceac79a05fe0249c1268165cdf359b5f9ddf7 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Tue, 11 Dec 2018 14:36:29 -0600
Subject: [PATCH] Implement PVP repo API

This is most likely temporary until we have a way to change dependencies
using Arcade, or Arcade supports the repo API itself.
---
 eng/Versions.props | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 947783183..0d9d78a4d 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -46,6 +46,9 @@
     <NUnit3TemplatesVersion>1.5.1</NUnit3TemplatesVersion>
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-26011-01</CLI_NETStandardLibraryNETFrameworkVersion>
   </PropertyGroup>
+  <!-- Override isolated build dependency versions with versions from Repo API. -->
+  <Import Project="$(DotNetPackageVersionPropsPath)"
+          Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
   <PropertyGroup>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
     <UsingToolSymbolUploader>true</UsingToolSymbolUploader>
-- 
2.18.0

